test/functional/rpc_zmq.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test for the ZMQ RPC methods."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+
+class RPCZMQTest(BitcoinTestFramework):
+
+    address = "tcp://127.0.0.1:28332"
+
+    def set_test_params(self):
+        self.num_nodes = 1
+        self.setup_clean_chain = True
+
+    def run_test(self):
+        self._test_getzmqnotifications()
```

Comment:
The test should be skipped similar to `interface_zmq.py`. Otherwise it will fail.

Reply:
That is indeed a good point!  Since this was already merged, should I create a new PR with this fix?

Reply:
I'll work on a fix and send in a PR with it as soon as I'm ready.  Thanks again for spotting this!

Reply:
Created #13646.